styles/SAP/Products.yml

```diff
@@ -0,0 +1,27 @@
+extends: substitution
+message: "Use '%s' instead of '%s'."
+link: 'https://www.sapbrandtools.com/naming-center/'
+level: error
+ignorecase: false
+swap:
+  (?:[^\s]*) ?(?:[^\s]*) ?AI SDKs?: SAP Cloud SDK for AI
```

Comment:
[fyi]: I have to remove `SAP Cloud SDK` with `SAP Cloud SDK for AI`, because otherwise it is hard for vale to decide which SDK we mean when it sees `SDK`. Also I feel using the official term `SAP Cloud SDK for AI` is somewhat better than `AI SDK`